src/tracing/service/tracing_service_impl.cc

```diff
@@ -4355,7 +4395,8 @@ base::Status TracingServiceImpl::FinishCloneSession(
     bool final_flush_outcome,
     std::optional<TriggerInfo> clone_trigger,
     base::Uuid* new_uuid,
-    int64_t clone_started_timestamp_ns) {
+    int64_t clone_started_timestamp_ns,
+    base::ScopedFile output_file_fd) {
```

Comment:
naming-wise I feel it's a bit easier to understand if you call this write_into_file_fd, so it's clear this is only valid if the session was a write-into-file itself.

Reply:
For me it seems to be more confusing to have two different things with the same name.
The `write_into_file` unambiguously names the session that have a file it may write to.
The `output_file_fd` names the file, only used for the clone operation. If we rename it to the `write_into_file_fd` it will be harder to figure out that this thing is relevant only for the clone operation.